File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");  //prompts human for desired height
    }
    while (0 >= height || height >= 9);  //ensures input is between 1 and 8
    
    for (int row = 0; row < height; row++)  //for loop for rows
    {
        for (int i = 0; i < (height - row - 1); i++)  //prints spaces
        {
            printf(" ");
        }
        for (int j = 0; j < row + 1; j++)  //prints first set of hashes
        {
            printf("#");
        }
        printf("  ");  //prints gap
        for (int k = 0; k < row + 1; k++)
        {
            printf("#");  //prints second set of hashes
        }
        printf("\n");  //goes to next line
    }
}

File: mario.py
from cs50 import get_int

while True:
    n = get_int("Height: ")
    if n >= 1 and n <= 8:
        break  # exits loop if n is between 1 and 8, inclusive

for i in range(n):
    for j in range (n - i - 1):  # prints spaces before pyramid
        print(" ", end="")

    for k in range(i + 1):  # loop that prints first pyramid
        print("#", end="")

    print("  ", end="")  # prints gap between pyramids

    for l in range(i + 1):  # loop that prints second pyramid
        print("#", end="")

    print("")  # new line